import-export-cli/integration/apim/apim.go

```diff
@@ -2039,6 +2049,225 @@ func (instance *Client) registerClient(username string, password string) dcrResp
+
+	base.SetDefaultRestAPIHeaders(instance.accessToken, request)
+
+	base.LogRequest("apim.AddThrottlePolicy()", request)
+
+	response := base.SendHTTPRequest(request)
+
+	defer response.Body.Close()
+
+	base.ValidateAndLogResponse("apim.AddThrottlePolicy()", response, 201)
+
+	json.NewDecoder(response.Body).Decode(&throttlePolicyResponse)
+
+	return throttlePolicyResponse
+}
+
+// DeleteThrottlePolicy : Deletes Throttling Policy from APIM using UUID
+func (instance *Client) DeleteThrottlePolicy(policyID, policyType string) {
+
+	policiesURL := instance.adminRestURL + "/throttling/policies/" + policyType + "/" + policyID
+
+	request := base.CreateDelete(policiesURL)
+
+	base.SetDefaultRestAPIHeaders(instance.accessToken, request)
+
+	base.LogRequest("apim.DeleteThrottlePolicy()", request)
+
+	response := base.SendHTTPRequest(request)
+
+	defer response.Body.Close()
+
+	base.ValidateAndLogResponse("apim.DeleteThrottlePolicy()", response, 200)
+}
+
+// GetThrottlePolicy : Get Throttle Policy from APIM using UUID
+func (instance *Client) GetThrottlePolicy(policyID, policyType string) map[string]interface{} {
+	var policyResponse map[string]interface{}
+
+	policiesURL := instance.adminRestURL + "/throttling/policies/" + policyType + "/" + policyID
+
+	request := base.CreateGet(policiesURL)
+
+	base.SetDefaultRestAPIHeaders(instance.accessToken, request)
+
+	base.LogRequest("apim.GetThrottlePolicy()", request)
+
+	response := base.SendHTTPRequest(request)
+
+	defer response.Body.Close()
+
+	base.ValidateAndLogResponse("apim.GetThrottlePolicy()", response, 200)
+
+	json.NewDecoder(response.Body).Decode(&policyResponse)
+
+	return policyResponse
+}
+
+// GetThrottlePolicyID : Get Throttle Policy UUID using policy name from APIM
+func (instance *Client) GetThrottlePolicyID(t *testing.T, username, password, policyName, policyType string) string {
+	var policyListResponse utils.ThrottlingPoliciesDetailsList
+	var uuid string
+
+	instance.Login(username, password)
+
+	queryType := ""
+	switch policyType {
+	case ApplicationThrottlePolicyType:
+		queryType = applicationThrottlePolicyQuery
+	case CustomThrottlePolicyType:
+		queryType = customThrottlePolicyQuery
+	case AdvancedThrottlePolicyType:
+		queryType = advancedThrottlePolicyQuery
+	case SubscriptionThrottlePolicyType:
+		queryType = subscriptionThrottlePolicyQuery
+	}
+
+	getPoliciesURL := instance.adminRestURL + "/throttling/policies/search/?query=type:" + queryType
+
+	request := base.CreateGet(getPoliciesURL)
+
+	base.SetDefaultRestAPIHeaders(instance.accessToken, request)
+
+	base.LogRequest("apim.GetThrottlePolicyID()", request)
+
+	response := base.SendHTTPRequest(request)
+
+	defer response.Body.Close()
+
+	base.ValidateAndLogResponse("apim.GetThrottlePolicyID()", response, 200)
+
+	err := json.NewDecoder(response.Body).Decode(&policyListResponse)
+	if err != nil {
+		t.Fatal(err)
+	}
+	throttlePolicyList := policyListResponse.List
+	for i := range throttlePolicyList {
```

Comment:
use `for key, val := range`. You don't have to use the index again

Reply:
Fixed with 120973b08a729a32ee8bf155f3c53e52a462756b